api-dummy/data/articles/0065-1737-azm-31-03-0367/1/en/front.xml

```diff
@@ -1,4 +1,5 @@
 <?xml version="1.0" ?>
+<?xml-model href="http://localhost:8081/schemas/scielo/article/front.rng"?>
```

Comment:
big assumption, would something like `/schemas/...` work?

Reply:
Perhaps it can be interpolated by the api-dummy when serving the file, with a default of `http://localhost:8081`

Reply:
Stops Oxygen from being able to validate the files locally. Going to see if I can set up a mapping there somehow.